src/Rcl/Options.hs

```diff
@@ -0,0 +1,35 @@
+{-# LANGUAGE StrictData #-}
+
+module Rcl.Options
+  ( Options(..)
+  , parseOptions
+  )
+where
+
+import RIO
+
+import Data.Set (Set)
+import qualified Data.Set as Set
+import Options.Applicative
+import Rcl.PackageName
+import Rcl.Resolver
+
+data Options = Options
+  { oDebug :: Bool
+  , oFromResolver :: Resolver
+  , oToResolver :: Resolver
+  , oDependencies :: Set PackageName
```

Comment:
I would be awesome if you could use the `stack.yaml` to source `package.yaml` files and pull dependencies from there. Shouldn't be too hard to do.

Reply:
I did start with that intention, but as soon as you need to be generalized or flexible to possible project setups, it quickly spirals. It was *much* easier to leave it to the caller to do something like this: https://github.com/freckle/megarepo/blob/master/backend/scripts/dependencies

Reply:
I could add this small shell snippet to the `README` though.